SuperBuild/External_CTK.cmake

```diff
@@ -81,7 +81,7 @@ if(NOT DEFINED CTK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
   ExternalProject_Add(${proj}
     ${${proj}_EP_ARGS}
     GIT_REPOSITORY "${Slicer_${proj}_GIT_REPOSITORY}"
-    GIT_TAG "${Slicer_${proj}_GIT_TAG}"
+    GIT_TAG visualDICOMBrowserENH
```

Comment:
This branch is still necessary for this PR? Or no since Visual DICOM browser stuff has since been integrated?

Reply:
no it is not needed. I mean it was actually never needed for this branch. I just fixed it right now